src/com/ableton/Pyenv.groovy

```diff
@@ -48,7 +48,7 @@ class Pyenv implements Serializable {
     script.withEnv(["PYENV_VERSION=${pythonVersion}"]) {
       List installCommands = [
         "export PYENV_ROOT=${pyenvRoot}",
-        "export PATH=\$PYENV_ROOT/bin:\$PATH",
+        "export PATH=${pyenvRoot}/bin:\$PATH",
```

Comment:
In 2a9caf3ab781cbc9575271a6a96b65314ebcd2a3, I don't see why we need to change this line.

Reply:
See 4270827.